src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1976,16 +1976,18 @@ public void testTryResourcesNotStrict() throws Exception {
      */
     @Test
     public void testArgumentOrderOfErrorMessages() {
-        final String[] arguments = {"##0##", "##1##", "##2##"};
+        final Object[] arguments = {"##0##", "##1##", "##2##"};
         final String[] messages = {
-            getCheckMessage(MSG_ERROR, (Object[]) arguments),
```

Comment:
Redundant type cast. This test is rewritten to use `Object[]` instead of `String[]`